homeassistant/components/holiday/__init__.py

```diff
@@ -20,6 +20,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Holiday from a config entry."""
     country: str = entry.data[CONF_COUNTRY]
     province: str | None = entry.data.get(CONF_PROVINCE)
+    categories: list[str] | None = entry.data.get(CONF_CATEGORIES)
```

Comment:
Here you are reading the province and the categories from `entry.data`, but I understood that they were moved to options instead.

Reply:
Only categories are in options. But this is only used for pre-loading the library so therefore wasn't affecting anything.